README.md

```diff
@@ -70,6 +71,7 @@ A curated list of awesome Kotlin frameworks, libraries, documents and other reso
 - [Building APIs on the JVM Using Kotlin and Spark](http://nordicapis.com/building-apis-on-the-jvm-using-kotlin-and-spark-part-1/)
 - [Using Project Kotlin for Android](https://docs.google.com/document/d/1ReS3ep-hjxWA8kZi0YqDbEhCqTt29hG8P44aA9W0DM8/edit)
 - [Kotlin for cross platform mobile app development](https://robovm.com/kotlin-for-cross-platform-mobile-app-development/)
+- [『Android 还可以这样开发』](http://zhuanlan.zhihu.com/kotandroid)
 
```

Comment:
Please move this line to the [Others](https://github.com/mcxiaoke/awesome-kotlin#others) section and do not using Chinese , because not all people's computer can display Chinese characters, thanks.